surveys/urls.py

```diff
@@ -10,4 +10,9 @@
         views_api.SurveyPostView.as_view(),
         name='survey-post',
     ),
+    url(
+        r'^forms/(?P<pk>\d+)/respond/(?P<user_id>\w+)?$',
```

Comment:
The `w` at least seems to accept underscores (going by the URL test), so I figure it should be fine.